src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -70,7 +95,7 @@ public static TimeSpan CalculateBackoffDuration(this TimeSpan minDuration, TimeS
 
             //
             // IMPORTANT: This can overflow
-            double calculatedMilliseconds = Math.Max(1, minDuration.TotalMilliseconds) * ((long)1 << Math.Min(attempts, MaxAttempts));
+            double calculatedMilliseconds = minDuration.TotalMilliseconds * (1L << Math.Min(attempts, MaxSafeShiftBit));
```

Comment:
Let's keep the original `Math.Max` calculation because `minDuration.TotalMilliseconds` can still be less than 1 ms. We dont want to reduce the calculated exponential by multiplying it with a number less than 1.

Reply:
Sounds good.